modules/core/src/main/scala/com/romanidze/kaesquella/core/models/ExecutionError.scala

```diff
@@ -0,0 +1,45 @@
+package com.romanidze.kaesquella.core.models
```

Comment:
nit: I usually prefer to keep all utility case classes in one file. It simplifies reading the code.